scripts/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04
 
 ENV DEBIAN_FRONTEND noninteractive
 
 COPY . .
 
 RUN apt update && apt dist-upgrade -y
-RUN ./configure.sh
+RUN ./install-build-tools.sh
```

Comment:
Before running a build by `./scripts/build.sh`, I manually ran `./install-build-tools.sh` and `./setup-dependencies.sh` in the command line. 
Adding these bash script calls in the Dockerfile saves time.